### DISCUSSION TITLE
Max Concurrent Connections

### DISCUSSION THREAD

**Discussion by chattouti** [postalserver/postal#2369] [Help with using Postal]
There are ESPs (Email Service Providers) that only accept a limited number of simultaneous connections, such as Orange France, which only accepts 3 simultaneous connections. How to solve this problem with Postal?

**Comment by willpower232** [comment]
Postal retries messages up at 18 times by default but depending on how many messages you're sending to one particular company with these limits and how many workers you have this may or may not be enough.
